packages/abstract-sdk/Cargo.toml

```diff
@@ -11,7 +11,7 @@ resolver = "2"
 
 # for quicker tests, cargo test --lib
 [features]
-default = []
```

Comment:
We don't want to default this

Reply:
yea sorry i left it on again...

Reply:
You don't have to turn it on, you can enable the --all-features in your IDE so it will analyze everything

Reply:
thanks for the tip!